cartoframes/data/dataset/dataset_info.py

```diff
@@ -9,6 +9,9 @@
 from ...utils.geom_utils import setting_value_exception
 from ...utils.columns import normalize_name
 
+from warnings import filterwarnings
```

Comment:
Is that still necessary after removing warnings in CARTO Python?

Reply:
Yes, it is. In this case, to get the info about datasets from a CARTO account (we need to get the privacy), we are using a non public API. In the carto-python code we are raising a `FutureWarning` warning about that. This is what we are filtering here.

Reply:
By the way, we have some of the spread by the CARTOframes code